2200048/문제3.py

```diff
@@ -0,0 +1,10 @@
+# 3번
+T = int(input())
+string = {'b' : 'd', 'd' : 'b', 'p' : 'q', 'q' : 'p'}
+for i in range(1,T+1):
+    out = []
+    mir = list(input())
+    mir = sorted(mir,reverse=True)
```

Comment:
```suggestion
    mir = mir[::-1]
```
`sorted`는 정렬하는 함수입니다! 다른 함수를 쓰거나, 위와 같은 방법으로 문자열을 뒤집을 수 있습니다. 😄